### PR TITLE
Added possibility to customize labels throughout the application

### PR DESCRIPTION
**Pull request by laudibert**
PR for #116.

Added possibility to configure all button titles, form titles and placeholders throughout the application. Also added the possibility to choose specific form titles within a page as well as a specific "+ Add Item" button text.

### CODE REVIEW

**Comment by laudibert** [comment]
Anything wrong with that one @dsternlicht ? If there is anything that need changes / restructuring let me know.

I unfortunately found a miss in the configuration schema I did while updating that branch, it was missing the `customActions` part. I fixed it here but it can obviously be fixed on a fix branch of its own.

**Comment by dsternlicht** [comment]
> Anything wrong with that one @dsternlicht ? If there is anything that need changes / restructuring let me know.
> 
> I unfortunately found a miss in the configuration schema I did while updating that branch, it was missing the `customActions` part. I fixed it here but it can obviously be fixed on a fix branch of its own.

What do you think about using the same property and interface `customLabels` for pages? Then it would be nice to add inheritance from top level.

**Comment by laudibert** [comment]
I did that at first. But then I thought it seemed quite absurd to have different placeholders default values in different pages. There is already a way to customize placeholders for each fields which seemed enough.

That being said, it does seem cleaner to have the same object. For now the inheritance is already taken into account though, a custom label on a page will be taken first, then look if there is a global one.

Up to you, I can reverse it to the same object.

**Comment by dsternlicht** [comment]
@laudibert IMHO, consistency is KING :)
Let's go with the same object.

**Comment by laudibert** [comment]
> @laudibert IMHO, consistency is KING :)

Yeah, you're right.

It's done.

File: src/assets/schemas/config.schema.json [hunk 1]
```diff
@@ -505,6 +501,9 @@
         },
         "placeholders": {
           "$ref": "#/definitions/placeholderLabels"
+        },
+        "tableColumnHeaders": {
```
Comment (dsternlicht) [inline src/assets/schemas/config.schema.json:505]: What's the tableColumnHeaders? where does it renders? Is it the title of the actions modal?

Comment (laudibert) [reply src/assets/schemas/config.schema.json:505]: Yes. I made it that if by any chance one day another generic column gets added.

**Comment by dsternlicht** [comment]
Merged!

**Pull request merged**